js/forum/src/components/EmojiAreaButton.js

```diff
@@ -8,23 +8,18 @@ import icon from "flarum/helpers/icon";
 
 export default class EmojiAreaButton extends Component {
 
-    init() {
-        this.textEditor = null;
```

Comment:
Oh I see. Okay I'll change that (even if this is probably the only place this component will ever be used). Thanks !